src/foremast/pipeline/construct_pipeline_block_lambda.py

```diff
@@ -68,6 +68,11 @@ def construct_pipeline_block_lambda(env='',
     instance_security_groups.append(gen_app_name)
     instance_security_groups.extend(settings['security_group']['instance_extras'])
 
+    for each_sg, duplicate_sg_name in SECURITYGROUP_REPLACEMENTS.items():
```

Comment:
Third copy means we really need a function.

Reply:
Done. Moved to utils.